packages/grafana-ui/src/components/Drawer/Drawer.story.tsx

```diff
@@ -13,12 +13,25 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disable: true,
+    },
+  },
+  args: {
+    closeOnMaskClick: true,
+    scrollableContent: false,
+    width: '40%',
+    expandable: false,
+    subtitle: 'This is a subtitle.',
+  },
+  argTypes: {
+    title: { control: { type: 'text' } },
+    width: { control: { type: 'text' } },
+    subtitle: { control: { type: 'text' } },
   },
 };
 
-export const global = () => {
-  const drawerTitle = text('title', 'Drawer title');
-
+export const Global: Story = (args) => {
```

Comment:
```suggestion
export const Global: Story<Props> = (args) => {
```
I think it's a good idea to pass the component's props to Story type for better typechecking, otherwise args type will be `[key: string]: any;`, which isn't too helpful.